src/main/java/pl/wedding/management/employee/model/StatusEmployee.java

```diff
@@ -0,0 +1,5 @@
+package pl.wedding.management.employee.model;
+
+public enum StatusEmployee {
+    WORKING, OFF, SICK, FIRED
```

Comment:
Fired may not be only reason of leaving work. there might be as well 'LEFT', or something indicating leaving company (or is it OFF?)

Reply:
about this OFF i thinking about the employer is on vacation or like this :) and I don't know how name this situation. Maybe like this would be ok  (WORKING, VACATION, SICK, OFF)?

Reply:
I think at this moment we should have at least those four (WORKING, VACATION, SICK, OFF). When situation will change in the future, new statuses should be added.

Reply:
But worth to add, that I'm not to decide if this is enough :) I'm only curious here. Ask @wedding-hall-adm if those are enough.